index.js

```diff
@@ -26,7 +24,7 @@ var styles = StyleSheet.create({
     },
 
     absolute: {
-        position: "absolute",
+        position: "fixed",
```

Comment:
Sometimes modal box are used inside a React component. Making position to "fixed", removes the flexibility. This seems unnecessary.

Reply:
changed to absolute again.